code_style.md

```diff
@@ -164,6 +158,14 @@ ECMAScript
 - Apart from that, newer ES features should be used whenever the author deems them to be appropriate.
 - Flow annotations are welcome and encouraged.
```

Comment:
This should be probably be edited to encourage Typescript or deleted.

Reply:
Thanks I added a reference to TypeScript